web/src/features/manual-scoring/lib/aggregateScores.ts

```diff
@@ -0,0 +1,104 @@
+import { type APIScore } from "@/src/features/public-api/types/scores";
+import { type ScoreSource, type ScoreDataType } from "@langfuse/shared";
+
```

Comment:
new naming (qual, quant), could we stick with numeric, categorical and include boolean in categorical?